docs/reference/mapping/params/meta.asciidoc

```diff
@@ -29,3 +29,24 @@ than or equal to 50.
 
 NOTE: Field metadata is updatable by submitting a mapping update. The metadata
 of the update will override the metadata of the existing field.
+
+Elastic products standardize on the following metadata, which you may use too
+in order to get a better out-of-the-box experience with your data.
+
+unit::
+
+  The unit of the field. Recognizes `"byte"` as well as
+  <<time-units,time units>>. By default, fields are assumed to not have a unit.
+  This information should only be specified on numeric fields.
+
+metric_type::
+
+  The type of metric. Recognizes `"gauge"` and `"counter"`. By default, fields
+  are assumed to not be a metric. This information should only be specified on
+  numeric fields.
+
+ratio::
+
+  Whether the field is a ratio. Recognizes `true` and `false` (default). The
```

Comment:
```suggestion
  Whether the field is a ratio or percent. Recognizes `true` and `false` (default). The
```

Reply:
My understanding was that we always use ratios in the data we produce, never percentages (ie. ratio * 100), is this assumption wrong? If we have both, do we need to be able to make the distinction via this metadata so that Kibana would know?

Reply:
@roncohen @ruflin and I had a small discussion about this yesterday. We see how this can be confusing for users, so we agreed this could be the best approach:

Having only `percent` stored as a unit (no need for a ratio flag in meta), where the value is expected to be in the [0-1] notation. This is how Metricbeat stores percentages today.

WDYT?